cg/cli/workflow/mip_rna/store.py

```diff
@@ -34,90 +38,29 @@ def analysis(context, config_stream):
 
     if not config_stream:
         LOG.error("provide a config, suggestions:")
-        for analysis_obj in context.obj["tb_api"].analyses(
-            status="completed", deleted=False
-        )[:25]:
+        for analysis_obj in tb_api.analyses(status="completed", deleted=False)[:25]:
```

Comment:
I guess we want only those analyses ran with mip-rna from trailblazer, how do think about that?

Reply:
Resolved in https://github.com/Clinical-Genomics/cg/pull/570/commits/f8fb464ff2b8eb29b14b956902d805a4e59392e8